pkg/arvo/lib/graph.hoon

```diff
@@ -76,6 +76,7 @@
 ++  get-graph
   |=  res=resource
   ^-  update:store
+  =-  -(p *time)
```

Comment:
So is this timestamp not used for anything? If it is, then we have to be careful about use of it not overlapping with results coming from this function. How would it having an ancient/bogus value affect behavior, if it's used?